cds_ils/importer/providers/cds/rules/book.py

```diff
@@ -6,73 +6,3 @@
 # the terms of the MIT License; see LICENSE file for more details.
 
 """CDS-ILS MARCXML book rules."""
-
```

Comment:
why are these functions removed?

Reply:
Because they already existed in the `base.py` and they were not overriding them. Same for languages function in journal rules.

Reply:
you can remove the title and number of pages, but there is some kind of confusion with alternative_titles rule, please leave it as it is, I will take a look on how to solve it

Reply:
what about languages from the journal rules?

Reply:
this rule is the same, so you can remove it indeed. nice catch for the repeated code!